src/common-elements/_discussions.scss

```diff
@@ -1,22 +1,25 @@
+body.discussion .content-wrapper {
+  padding-top: 0;
+}
 .a--courseware-top-nav__list {
-
-  li.right {
-    margin-left: auto;
-  }
+  list-style-type: none;
 }
+.a--container {
+  .discussion-submit-post:not(:disabled),
+  .new-post-btn:not(:disabled) {
+    border-radius: $buttons-border-radius;
+    border: 0.1rem solid $brand-primary-color;
+    padding: 1rem 1.5rem;
+    font-size: 1.4rem;
+    @include transition(all 0.3s ease-in-out);
+    opacity: 1;
 
-a.new-post-btn {
-  border-radius: $buttons-border-radius;
-  border: 0.1rem solid $brand-primary-color;
-  padding: 1rem 1.5rem;
-  @include transition(all 0.3s ease-in-out);
-  opacity: 1;
-
-  span {
-    margin-right: 0.5rem;
-  }
-
-  &:hover, &:focus {
-    opacity: 0.6;
+    // Have to specify :not:(:disabled) above to win against
+    // _edx-overrides.scss
```

Comment:
Is there a nicer way to do this, like adding `html body` first? Or is `:not(:disabled)` more efficient? Also, the comment has a typo (extra `:`)